helm/alfresco-content-services/values.yaml

```diff
@@ -21,10 +21,27 @@ repository:
     pullPolicy: Always
     internalPort: 8080
     hazelcastPort: 5701
-  initContainer:
-    image:
-      repository: busybox
-      tag: latest
+  initContainers:
+    db:
+      image:
+        repository: busybox
+        tag: 1
```

Comment:
It looks like we've got two different approaches in the charts, sometimes we split the repository and tag and sometimes not i.e. "busybox:1"

Reply:
yes, fixed the other occurences also, we will have repository:tag everywhere